src/utils/visning.tsx

```diff
@@ -31,6 +34,14 @@ export const verdiTilTekstsvar = (
       </ul>
     );
   } else if (typeof verdi === 'string') {
+    try {
+      if (isValidISODateString(verdi)) {
+        const formattertDato = formatDate(strengTilDato(verdi));
+        return <Normaltekst>{formattertDato}</Normaltekst>;
+      }
+    } catch (e) {
+      return <Normaltekst>{verdi}</Normaltekst>;
```

Comment:
Hvis isValidISODateString er false er eneste konsekvens at vi skriver ut som streng? Burde jo være greit? :)

Reply:
Ja, vi fallbacker til å skrive ut streng.
Problemet med `isValidISODateString(...)` er at den kaster en del exceptions hvis input er koko nok - så la en try/catch rundt for å unngå at vi smeller når en verdi f.eks. ikke er satt.

Reply:
Ja, denne funksjonen tar strenger, datoer og boolean og skriver det ut. Nå som datoer er strenger, er dette måten vi finner om strengen er dato eller streng :)

Reply:
Woops, så ikke svaret til Charlie før etter jeg kommenterte